protocol/transport/whisper_service.go

```diff
@@ -187,35 +185,24 @@ func (a *WhisperServiceTransport) Request(ctx context.Context, options RequestOp
 }
 
 func (a *WhisperServiceTransport) selectAndAddMailServer() (string, error) {
+	var node string
```

Comment:
```suggestion
	var enodeAddr string
```

to avoid confusion with `a.node`.